src/server/services/wfs/qgswfsparameters.h

```diff
@@ -134,8 +134,7 @@ namespace QgsWfs
         NONE,
         GML2,
         GML3,
-        GeoJSON,
-        XSD
```

Comment:
Is the removal of XSD related? I see it's not used anywhere so it was probably a no-op anyway.

Reply:
The removal is under a specific commit, and I do it because I found it is not used anywhere. I dont think it needs a specific pull request.